sql/training_cms_setup/mysql4-install-0.1.0.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/** @var Mage_Core_Model_Resource_Setup $installer */
+$installer = $this;
+$installer->startSetup();
+
+
+/**
+ * Creating a table training_cms_page
+ */
+$installer->run("

```

Comment:
Rewrite this code with the usage of Magento approach of adding/editing DB tables (see part 9 of "Install script (and two sample models)" and part 2 of "Install upgrade script" here http://inchoo.net/magento/magento-install-install-upgrade-data-and-data-upgrade-scripts/.